pkgs/racket-doc/scribblings/raco/test.scrbl

```diff
@@ -179,15 +179,19 @@ The @exec{raco test} command accepts several flags:
         as a whitespace-delimited list of arguments to add. To specify
         multiple arguments using this flag within a typical shell,
         @nonterm{arguments} must be
-        enclosed in quotation marks.
+        enclosed in quotation marks.}
+
+ @item{@DFlag{output} or @Flag{o} @nonterm{filename}
+       --- Save all stdout and stderr output into @nonterm{filename}.
  }
 ]
 
 @history[#:changed "1.1" @elem{Added @DFlag{heartbeat}.}
          #:changed "1.4" @elem{Changed recognition of module suffixes to use @racket[get-module-suffixes],
                                which implies recognizing @filepath{.ss} and @filepath{.rkt}.}
          #:changed "1.5" @elem{Added @DPFlag{ignore-stderr}.}
-         #:changed "1.6" @elem{Added @DPFlag{arg} and @DPFlag{args}.}]
+         #:changed "1.6" @elem{Added @DPFlag{arg} and @DPFlag{args}.}
+         #:changed "1.7" @elem{Added @DFlag{output} and @Flag{o}.}]
```

Comment:
Which revison should be refered in this history? the racket version? 7.9.0.3? @pmatos

Reply:
It should be 1.8. It should correspond to https://github.com/racket/racket/blob/master/pkgs/compiler-lib/info.rkt.